wikipron.py

```diff
@@ -9,12 +9,19 @@
 
 from typing import Callable, Iterator, List, Optional, TextIO, Tuple
 
-import iso639
-import requests
-import requests_html
+try:
+    import iso639
+    import requests
+    import requests_html
+except ModuleNotFoundError:
+    logging.warning(
+        "Thirty-party packages are not imported. "
+        "This situation should arise only in the installation phase, "
+        "where setup.py runs and imports this present module."
+    )
```

Comment:
Not super pretty, but the (an?) alternative would be to go with the usual full-blown package directory structure:

```
setup.py  # can read in _version.py without issues
wikipron/  # this directory would be in the site-packages, with _version.py inside
    __init__.py
    _version.py  # with only` __version__ = "0.1.1"`
    scrape.py
    config.py
    cli.py
tests/
    ...
```

which would seem to be a little bit too much about refactoring than fixing the import bug? I'm on the fence.

Reply:
Let's do it later if you want.